sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStore.scala

```diff
@@ -509,4 +549,3 @@ object StateStore extends Logging {
     }
   }
 }
-
```

Comment:
nit for the last empty line.

Reply:
This is actually removing the double empty lines, but doesn't matter with keeping it.